website/server/controllers/api-v3/members.js

```diff
@@ -57,6 +57,288 @@ api.getMember = {
   },
 };
 
+function _addAchievement(achievementsResult, type, key, title, text, icon, category, value, earned) {
```

Comment:
yeah, I'll change it, but it would also increase wrongly used parameters, don't you think?